app/models/tweet.rb

```diff
@@ -73,7 +73,7 @@ def make_tweet_from_api_object(tweeted)
       tweet = Tweet.where(twitter_id_code: tweeted.attrs[:id_str]).first_or_initialize
       tweet.twitter_uid = tweeted.user.id.to_s
       tweet.twitter_username = tweeted.user.screen_name.downcase
-      tweet.user_id = User.find_by(twitter_username: tweeted.user.screen_name).try(:id)
+      tweet.user_id = User.find_by(twitter_username: tweeted.user.screen_name)&.(:id)
```

Comment:
Btw it can be a bit shorter with `User.find_by(twitter_username: tweeted.user.screen_name)&.id`